deis/manifests/deis-logger-fluentd-daemon.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: extensions/v1beta1
```

Comment:
do we need to install any extra extensions in order for daemonsets to work in k8s v1.1 or is this ready OOTB? Just wondering if there needs to be additions to the installation docs.